soroban-env-common/src/env.rs

```diff
@@ -152,6 +152,15 @@ macro_rules! call_macro_with_all_host_functions {
                 /// running contract. Traps if the running contract was not
                 /// invoked by a contract.
                 {"4", fn get_current_contract() -> Object }
+
+                /// Return the protocol version of the current ledger as a u32.
+                {"5", fn get_ledger_version() -> RawVal }
```

Comment:
I'm suspicious of whether we should include a version, given https://github.com/stellar/rs-soroban-env/issues/312#issuecomment-1213508126. cc @graydon @MonsieurNicolas